site/sku.routes.js

```diff
@@ -61,5 +61,11 @@ module.exports = [
       { route: `/css/${name}/releases` },
     ]),
   ),
-  ...iconNames.map((name) => ({ route: `/components/${name}`, name })),
+  ...flatten(
+    iconNames.map((name) => [
+      { route: `/components/${name}`, name },
+      { route: `/components/${name}/props` },
+      { route: `/components/${name}/releases` },
+    ]),
+  ),
```

Comment:
I think it's safe to use the native [`.flat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat) nowadays (worth doing for the other usages in this file too).

Reply:
Probably even just [`flatMap`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap)